test/core/parser/test_convert.py

```diff
@@ -78,7 +78,10 @@ def testReal(self):
         self.check("1.5`", Real("1.5"))
         self.check("0.0", Real(0))
         self.check("-1.5`", Real("-1.5"))
-
+        self.check("0`3", Integer(0))
+        self.check("0``3", "0.000`3")
```

Comment:
Why this one contains 2 "`"?

Reply:
2 "`" represents an accuracy on 2, meaning that the number has an absolute uncertainty dx=0.01

Reply:
An `Integer` 0 with an absolute uncertainty is represented as a `PrecisionReal` with that uncertainty, while if the uncertainty is relative, the absolute uncertainty vanish, and hence, it is an Integer.